File: 169-majority-element/majority-element.java
class Solution {
    public int majorityElement(int[] nums) {
       int el=0,ct=0;
       for(int i=0;i<nums.length;i++){
           if(ct==0){
               el=nums[i];
               ct++;
           }
           else if(nums[i]==el){
               ct++;
           }
           else{
               ct--;
           }
       }
       return el;
    }
}

File: 169-majority-element/majority-element.c
int majorityElement(int* nums, int numsSize) {
    int el = 0, ct = 0;
    for(int i = 0; i < numsSize; i++){
        if(ct == 0){
            el = nums[i];
            ct = 1;
        }
        else if(nums[i] == el){
            ct++;
        }
        else{
            ct--;
        }
    }
    return el;
}